templates/compose/labelstudio.yaml

```diff
@@ -0,0 +1,49 @@
+# documentation: https://labelstud.io/guide/
+# slogan: Label Studio is a multi-type data labeling and annotation tool with standardized output format
+# tags: workflow, orchestration, data-pipeline, python, labelstudio, ai, elasticsearch, datasets, data, machine-learning, data-science, nlp, images, vision
+# logo: svgs/labelstudio.png
+# port: 8080
+
+version: "3"
```

Comment:
Only for future reference, no longer needed for compose projects

Reply:
Got it. Gonna modify all of them.